src/OpenDebugAD7/TextPositionTuple.cs

```diff
@@ -41,6 +41,12 @@ public static TextPositionTuple GetTextPositionOfFrame(PathConverter converter,
                     Path = convertedFilePath,
                     Name = Path.GetFileName(convertedFilePath)
                 };
+                if (!File.Exists(convertedFilePath))
+                {
+                    // mark the stack frame to be skipped by default
```

Comment:
it can still be selected manually

Reply:
Can you share an image of the dialog of it demphasized?

Reply:
https://github.com/microsoft/vscode/issues/65025#issuecomment-453645164
![stack](https://user-images.githubusercontent.com/323878/51057399-5cf1ad00-159a-11e9-88ea-6bc837fe9bc9.png)